.github/labeler-issue-triage.yml

```
@@ -298,6 +298,8 @@ service/route53domains:
   - '((\*|-) ?`?|(data|resource) "?)aws_route53domains_'
 service/route53resolver:
   - '((\*|-) ?`?|(data|resource) "?)aws_route53_resolver_'
+service/route53recoveryreadiness:
+  - '((\*|-) ?`?|(data|resource) "?)aws_route53_recovery_readiness_'
```

We prefer the service part of the resource type name to be a single word (i.e. no - or _ characters) - this will aid with future automation.

```suggestion
  - '((\*|-) ?`?|(data|resource) "?)aws_route53recoveryreadiness_'
```